test/wh_test.c

```diff
@@ -46,12 +46,14 @@ int whTest_Unit(void)
 {
     printf("Enter unit tests\n");
     /* Component Tests */
+#if 0
```

Comment:
?  Are the busted or sumthin?

Reply:
nope, oversight on my end just to speed up tests, good catch

Reply:
fixed in latest commit